packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -6725,3 +6769,28 @@ class _EditableTextTapUpOutsideAction extends ContextAction<EditableTextTapUpOut
     // The default action is a no-op.
   }
 }
+
+/// A [TextSpan] that overrides the style of its children with a given
+/// [TextStyle].
```

Comment:
Note for posterity: perhaps we should consider making `InlineSpan` `sealed`. This would disallow customers from subclassing it and would make sure this trick works. Just a thought, this isn't a blocker for landing this PR.

(This would be a breaking change, we'd likely want to use something like [`@Deprecated.subclass`](https://api.dart.dev/dev/latest/dart-core/Deprecated/Deprecated.subclass.html).)